src/app/modules/content/knowledge-component/knowledge-component.component.css

```diff
@@ -0,0 +1,46 @@
+.mat-sidenav-container {
+  color: white;
+  background-color: #141414;
```

Comment:
Definitely need to handle this at a central location. Research angular theming best practices.

For example, the AE for arrange task is white
![image](https://user-images.githubusercontent.com/7092212/144804785-24dfbe57-0601-4ef4-bab7-4e706ea5599a.png)

We want to avoid c&p this into the whole app, so that we can support easy change of themes.

Also, we use prismjs to support syntax highlighting (which currently doesn't work for some reason), so we should get that working and support it's theming https://github.com/PrismJS/prism-themes